Tchap/Modules/Forward/ForwardCoordinator.swift

```diff
@@ -0,0 +1,139 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+protocol ForwardCoordinatorDelegate: class {
+    /**
+     Called once the coordinator finished forward process
+     
+     @param coordinator caller of this method
+     @param roomID ID of the room the event has been forwarded to
+     @param error instance of the error if an error occured. Nil otherwise
+     */
+    func forwardCoordinator(_ coordinator: ForwardCoordinatorType, didForwardTo roomID: String, error: Error?)
```

Comment:
This method is not used (no delegate is provided to the ForwardCoordinator in the current implementation)
which means the error (if any) is not handled.

I think we can remove this method from the delegate for the moment, and handle the potential error at the ForwardCoordinator level, by adding an `AlertErrorPresenter` (or  an `ErrorPresenter`). Don't you think so?